README.md

```diff
@@ -102,7 +102,32 @@ arenas.
 }
 ```
 
-Eventually [all `std` collection types will be parameterized by an
+#### Serde
+
+Adding the `serde` feature flag will enable transparent serialization for Vec.
+
+```toml
+[dependencies]
+bumpalo = { version = "3.9", features = ["collections", "serde"] }
+```
+
+```rust,ignore
+use bumpalo::{Bump, collections::Vec};
+
+// Create a new bump arena.
+let bump = Bump::new();
+
+// Create a `Box`
+let vec = Vec::new_in( &bump);
+vec.push(1);
+vec.push(2);
+
+// Serialize with serde_json
+assert_eq!(serde_json::to_string(&vec).unwrap(), "[1, 2]");
+```
+
+Eventually
+[all `std` collection types will be parameterized by an
```

Comment:
Can you leave the "Eventually ..." paragraph with the first part of the `### Collections` section, and have the `#### Serde` section follow after that paragraph?

Reply:
Actually, can we just add a sentence/paragraph about serde here, rather than a whole subsection, and then reference/link to a dedicated/deduplicated serde section down below. That way we won't need to mention serde in both the collections and box sections.

Reply:
Sorry this was a rebase error.